keyboards/epomaker/th66/info.json

```diff
@@ -0,0 +1,253 @@
+            {"matrix": [3, 2], "x": 40, "y": 32, "flags": 4},
+            {"matrix": [3, 3], "x": 55, "y": 32, "flags": 4},
+            {"matrix": [3, 4], "x": 70, "y": 32, "flags": 4},
+            {"matrix": [3, 5], "x": 84, "y": 32, "flags": 4},
+            {"matrix": [3, 6], "x": 99, "y": 32, "flags": 4},
+            {"matrix": [3, 7], "x": 114, "y": 32, "flags": 4},
+            {"matrix": [3, 8], "x": 129, "y": 32, "flags": 4},
+            {"matrix": [3, 9], "x": 143, "y": 32, "flags": 4},
+            {"matrix": [3, 10], "x": 158, "y": 32, "flags": 4},
+            {"matrix": [3, 11], "x": 173, "y": 32, "flags": 4},
+            {"matrix": [3, 13], "x": 196, "y": 32, "flags": 1},
+            {"matrix": [4, 14], "x": 206, "y": 48, "flags": 4},
+            {"matrix": [5, 12], "x": 176, "y": 64, "flags": 1},
+
+            {"matrix": [4, 0], "x": 9, "y": 48, "flags": 1},
+            {"matrix": [4, 2], "x": 33, "y": 48, "flags": 4},
+            {"matrix": [4, 3], "x": 48, "y": 48, "flags": 4},
+            {"matrix": [4, 4], "x": 62, "y": 48, "flags": 4},
+            {"matrix": [4, 5], "x": 77, "y": 48, "flags": 4},
+            {"matrix": [4, 6], "x": 92, "y": 48, "flags": 4},
+            {"matrix": [4, 7], "x": 106, "y": 48, "flags": 4},
+            {"matrix": [4, 8], "x": 121, "y": 48, "flags": 4},
+            {"matrix": [4, 9], "x": 136, "y": 48, "flags": 4},
+            {"matrix": [4, 10], "x": 151, "y": 48, "flags": 4},
+            {"matrix": [4, 11], "x": 165, "y": 48, "flags": 4},
+            {"matrix": [4, 13], "x": 185, "y": 48, "flags": 1},
+            {"matrix": [5, 13], "x": 191, "y": 64, "flags": 4},
+            {"matrix": [5, 14], "x": 206, "y": 64, "flags": 4},
+            {"matrix": [5, 15], "x": 220, "y": 64, "flags": 4},
+
+            {"x": 92, "y": 8, "flags": 2},
+            {"x": 62, "y": 8, "flags": 2},
+            {"x": 33, "y": 8, "flags": 2},
+            {"x": 11, "y": 16, "flags": 2},
+            {"x": 13, "y": 32, "flags": 2},
+            {"x": 17, "y": 52, "flags": 2},
+            {"x": 51, "y": 56, "flags": 2},
+            {"x": 84, "y": 56, "flags": 2},
+            {"x": 110, "y": 56, "flags": 2},
+            {"x": 147, "y": 52, "flags": 2},
+            {"x": 173, "y": 52, "flags": 2},
+            {"x": 198, "y": 52, "flags": 2},
+            {"x": 202, "y": 28, "flags": 2},
+            {"x": 206, "y": 8, "flags": 2},
+            {"x": 165, "y": 8, "flags": 2},
+            {"x": 136, "y": 8, "flags": 2},
+
+            {"matrix": [0, 0], "x": 0, "y": 0, "flags": 1},
+            {"matrix": [5, 0], "x": 2, "y": 64, "flags": 1},
+            {"matrix": [5, 1], "x": 20, "y": 64, "flags": 1},
+            {"matrix": [3, 14], "x": 224, "y": 20, "flags": 4},
+            {"matrix": [3, 15], "x": 224, "y": 36, "flags": 4}
+        ]
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"matrix": [0, 0], "w": 1, "x": 0, "y": 0},
```

Comment:
```suggestion
                {"matrix": [0, 0], "x": 0, "y": 0},
```
etc...

Reply:
Thanks for reviewing, i've pushed new commit to resolved those.